### PR TITLE
storage: _delete_metric_splits can now batch over multiple metrics

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: gnocchi/storage/ceph.py [hunk 1]
```diff
@@ -94,23 +94,27 @@ def _store_metric_splits(self, metric, keys_aggregations_data_offset,
...
+                )
+                for name in names:
+                    try:
+                        self.ioctx.remove_object(name)
+                    except rados.ObjectNotFound:
+                        # It's possible that we already remove that object and
+                        # then crashed before removing it from the OMAP key
+                        # list; then no big deal anyway.
+                        pass
+                self.ioctx.remove_omap_keys(op, names)
```
Comment (chungg) [inline gnocchi/storage/ceph.py]: just remove all the keys at the end outside of loop.

Comment (jd) [reply gnocchi/storage/ceph.py]: Can't, it's per metric

Comment (chungg) [reply gnocchi/storage/ceph.py]: I don't understand... they're keys. how would they not be unique?

Comment (jd) [reply gnocchi/storage/ceph.py]: They are stored on different objects, e.g. `self._build_unaggregated_timeserie_path(metric, 3)`.

Comment (chungg) [reply gnocchi/storage/ceph.py]: hm got it

**Comment by jd** [comment]
Fix Ceph signature

**Pull request merged**